esp/services/ws_workunits/ws_workunitsService.cpp

```diff
@@ -4023,14 +4024,22 @@ bool CWsWorkunitsEx::onWUDetailsMeta(IEspContext &context, IEspWUDetailsMetaRequ
         }
         resp.setStatistics(statistics);
 
-        StringArray attributes;
-        for (unsigned i=WuAttr::WaAll+1; i<WuAttr::WaMax; ++i)
+        IArrayOf<IEspWUDetailsMetaProperty> properties;
+        for (WuAttr attr=WuAttr::WaKind; attr<WuAttr::WaMax; ++((unsigned &)attr))
```

Comment:
An alternative to the cast on the ++ would be

    WuAttr & operator++(WuAttr & x) { x = (WuAttr)(x+1); return x; }

defined in wuattr.hpp.